pkg/apis/mongodb/v1/mongodb_types.go

```diff
@@ -25,6 +25,9 @@ const (
 	// LastVersionAnnotationKey should indicate which version of MongoDB was last
 	// configured
 	LastVersionAnnotationKey = "lastVersion"
+
+	// TLSRolledOutKey indicates if TLS has been fully rolled out
+	TLSRolledOutKey = "tlsRolledOut"
```

Comment:
Good one! I'm totally on board with doing this now. How would we go about migrating from the old `lastVersion` to `mongodb.com/v1.lastVersion` for existing clusters?

Reply:
I don't know if we need to worry about this. To be safe we could just do a `|| lastVersion`, but always set the new tag.

Reply:
I think you're right, we don't have to worry about this. The operator will think the version hasn't changed as the new annotation is not set. It will then get updated at the end of the reconciliation loop.

Fixed :)